### ISSUE TITLE
YOLO8s: good accuracy, but too many false positives on white space.  YOLO11s: fewer false positives, but bounding boxes are shifted toward the top-left while width/height remain accurate.

### ISSUE THREAD

**Issue by UjjwalDodeja2** [ultralytics/hub#1168]
### Search before asking

- [x] I have searched the HUB [issues](https://github.com/ultralytics/hub/issues) and [discussions](https://github.com/ultralytics/hub/discussions) and found no similar questions.

### Question

I am migrating my project from Yolov8s to Yolo11s to reduce the number of false positives. I am using the detection model for Yolov8 to detect hand written signatures in documents. I trained the model on my personal dataset containing around a thousand images. 

When trained on top of Yolov8, the model was able to detect most signatures, and very accurately provide the bounding boxes. However, it resulted in multiple false positives on whitespaces throughout the document. 

When I train the Yolo11 on the same dataset, it solves the problem with false positives, and provides better detection of signatures. Nevertheless, the bounding box returned by Yolo11 are leaned towards the top-left of the documents on a regular basis. The width and height of the boxes are very accurate like the Yolov8 but the x and y coordinates have an inconsistent error%.

**SetUp:**
Ubuntu 24.04.2 LTS (GNU/Linux 6.8.0-71-generic x86_64)
CUDA Version: 12.4
Ultralytics version: 8.3.172
Python: 3.11.12

I have shared my training code below:

<img width="496" height="249" alt="Image" src="https://github.com/user-attachments/assets/047c2ebe-16c8-49e6-a937-d30c99c009ba" />
<img width="496" height="249" alt="Image" src="https://github.com/user-attachments/assets/d8beedaf-0ccc-4e3c-9afd-847bdec3404e" />

**Code:** 
from ultralytics import YOLO
import torch

model = YOLO("yolo11s.pt", "detect")  # initialize model

device = torch.device('cuda')

results = model.train(
    data="/home/trike/LLM/OD-Training/train/dataset.yaml",
    epochs=30,
    imgsz=640,
    batch=64,
    device=device)  # train the model

results = model.val()  # evaluate model performance on the validation set

model.to(device)
results = model.export(format="torchscript", imgsz=640, nms=False, device='cuda')
#export the model to torch format
**
I have tried using different number of epochs for better results ranging from 30 to 150 but failed to improve. 

Questions:
1. Is this a known issue in YOLO11 fine-tuning?
2. Could it be caused by preprocessing or augmentation changes?
3. Any recommended workaround?

<img width="496" height="249" alt="Image" src="https://github.com/user-attachments/assets/7658aea4-a484-4d3a-9135-8aa56d897661" />
<img width="496" height="249" alt="Image" src="https://github.com/user-attachments/assets/5df6b53f-2705-46fb-aa03-c3f559f1698d" />

For reference, I have added my test results from Yolov8 and Yolo11.

**Comment by UltralyticsAssistant** [comment]
👋 Hello @UjjwalDodeja2, thank you for raising an issue about [Ultralytics HUB](https://hub.ultralytics.com) 🚀! This is an automated response to help you get started—an Ultralytics engineer will also follow up soon.

Please visit our [HUB Docs](https://docs.ultralytics.com/hub) to learn more:
- [**Quickstart**](https://docs.ultralytics.com/hub/quickstart). Start training and deploying YOLO models with HUB in seconds.
- [**Datasets: Preparing and Uploading**](https://docs.ultralytics.com/hub/datasets). Learn how to prepare and upload your datasets to HUB in YOLO format.
- [**Projects: Creating and Managing**](https://docs.ultralytics.com/hub/projects). Group your models into projects for improved organization.
- [**Models: Training and Exporting**](https://docs.ultralytics.com/hub/models). Train YOLOv5 and YOLOv8 models on your custom datasets and export them to various formats for deployment.
- [**Integrations**](https://docs.ultralytics.com/hub/integrations). Explore different integration options for your trained models, such as TensorFlow, ONNX, OpenVINO, CoreML, and PaddlePaddle.
- [**Ultralytics HUB App**](https://docs.ultralytics.com/hub/app). Learn about the Ultralytics App for iOS and Android, which allows you to run models directly on your mobile device.
    * [**iOS**](https://docs.ultralytics.com/hub/app/ios). Learn about YOLO CoreML models accelerated on Apple's Neural Engine on iPhones and iPads.
    * [**Android**](https://docs.ultralytics.com/hub/app/android). Explore TFLite acceleration on mobile devices.
- [**Inference API**](https://docs.ultralytics.com/hub/inference-api). Understand how to use the Inference API for running your trained models in the cloud to generate predictions.

If this is a 🐛 Bug Report, please provide a minimum reproducible example (MRE) so we can investigate efficiently:
- Steps to reproduce, including exact training/val/inference commands and any export commands you used
- A small sample of images and labels demonstrating the issue (e.g., 5–10 images) in YOLO format
- Screenshots or short videos showing the shifted boxes vs. expected results
- Your dataset.yaml (with paths sanitized if needed)
- Environment details and exact package versions
You can follow our guide here: [reproduce](https://docs.ultralytics.com/help/minimum_reproducible_example/) ✅

If this is a ❓ Question, please include as much detail as possible (dataset specifics, model variant, training settings, augmentations, image sizes, export/runtime details) so we can provide targeted guidance.

We try to respond to all issues as promptly as possible. Thank you for your patience! 🙏
